invites.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"accounts-service/auth"
+	"accounts-service/models"
+	accountsv1 "accounts-service/protorepo/noted/accounts/v1"
+	"context"
+
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type invitesAPI struct {
+	accountsv1.UnimplementedInvitesAPIServer
+
+	auth        auth.Service
+	logger      *zap.Logger
+	groupRepo   models.GroupsRepository
+	accountRepo models.AccountsRepository
+	inviteRepo  models.InvitesRepository
+}
+
+var _ accountsv1.InvitesAPIServer = &invitesAPI{}
+
+func (srv *invitesAPI) SendInvite(ctx context.Context, in *accountsv1.SendInviteRequest) (*accountsv1.SendInviteResponse, error) {
+	token, err := srv.authenticate(ctx)
+	if err != nil {
+		return nil, status.Error(codes.Unauthenticated, err.Error())
+	}
+
+	accountId := token.UserID.String()
+
+	_, err = srv.groupRepo.Get(ctx, &models.OneGroupFilter{ID: in.GroupId})
+	if err != nil {
+		return nil, status.Error(codes.InvalidArgument, "failed to get group from group_id")
+	}
```

Comment:
Don't forget to check that the sender is part of the group.

Reply:
Same question for how to access the API cause it will be the easiest way there